docs/guides/kds-sync-name.md

```diff
@@ -0,0 +1,186 @@
+# What is the name of the resource after KDS sync?
+
+## Old policies (i.e. TrafficRoute, TrafficPermission)
+
+* CRD scope: Cluster
+* Kuma scope: Mesh
+
+### Global(k8s/universal) -> Zone(k8s/universal)
+
+```
+nameInZone := "${truncate(nameInGlobal, 236)}-${hash(mesh, nameInGlobal)}"
+```
+
+Example: `my-policy -> my-policy-fvdw2c2wx44fv4xz`
+
+### Zone(k8s/universal) -> Global(k8s/universal)
+
+Not supported
+
+## New policies (i.e. MeshTrafficPermission, MeshHTTPRoute)
+
+* CRD scope: Namespace
+* Kuma scope: Mesh
+
+### Global(k8s) -> Zone(k8s)
+
+```
+nameInGlobal = trimNsSuffix(nameInGlobal)
```

Comment:
There's a ns suffix in Global k8s?

Reply:
To me on global k8s you wouldn't have nsSuffixes if they are created in k8s no?

Reply:
We have nsSuffixes only in k8s, Kuma core model doesn't know about "namespace" and that's why we encode it as suffix.